GitHubContentUtility/Helpers/JwtHelper.cs

```diff
@@ -0,0 +1,46 @@
+﻿// ------------------------------------------------------------------------------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------------------------------------------------------------------------------
+
+// Partially ported from: https://github.com/microsoftgraph/msgraph-samples-dashboard/blob/dev/SamplesDashboard/SamplesDashboard/Helper/JWTHelper.cs
```

Comment:
We should ideally replace copied code in SamplesDashboard with the nuget package as well, but it is not part of this work item.

Reply:
@MaggieKimani1

Reply:
@zengin yeah I'll look into that.